Sprint_7 Done

IvshinOS opened the pull request.

IvshinOS committed:
Sprint_7 Done

alesyaisachenko commented on src/test/java/apirequests/CourierRequest.java:
```diff
@@ -0,0 +1,52 @@
+                .header("Content-type", "application/json")
+                .body(courier)
+                .when()
+                .post ("/api/v1/courier");
```
Можно лучше :

"/api/v1/courier" -- лучше было вынести в final static переменную.

alesyaisachenko commented on src/test/java/scootertests/TestCreateCourier.java:
```diff
@@ -0,0 +1,76 @@
+
+import static org.hamcrest.Matchers.*;
+
+public class TestCreateCourier extends ApiBase {
```
Это нужно обязательно исправить:

 TestCreateCourier extends ApiBase -- странно читать, что тестовый класс является наследником ApiКласса.

Тут либо нужно переименовать класс ApiBase и оставить текущее наследование, или классы CourierRequest и OrderRequest наследовать от ApiBase .

alesyaisachenko commented on src/test/java/scootertests/TestCreateCourier.java:
```diff
@@ -0,0 +1,76 @@
+        courierRequest.createCourier()
+                .then().assertThat().body("ok", is(true))
+                .and()
+                .statusCode(201);
```
Можно лучше:

статус коды можно было взять из библиотеки:
import  static  org.apache.http.HttpStatus.*;

statusCode(SC_CREATED)

IvshinOS committed:
1.Endpoints вынесены в final static
2.Изменены наследники класса ApiBase
3.Статус коды взять из библиотеки HttpStatus
